config/k8s-galera.txt

```diff
@@ -0,0 +1,21 @@
+# [This file is part of the zabbix-agent-osso package]
+
+# k8s-galera
+
+The k8s galera monitoring script monitors galera clusters setup with the mariadb operator for kubernetes.
+The script gets its status information from the mariadb cluster CRD inside your kubernetes cluster.
+
+For more information about the mariadb kubernetes operator, visit the [project homepage](https://github.com/mariadb-operator/mariadb-operator)
+
+Before setting up galera monitoring make sure your hosts have the following packages installed:
+
+- `cut`
+- `jq`
+- `kubectl`
```

Comment:
Technically these are not all packages. `cut` is in coreutils:
```console
$ dpkg -S `which cut`
coreutils: /usr/bin/cut
```

But I can turn a blind eye to this one.